postgresql-transactional.cabal

```diff
@@ -17,7 +23,7 @@ source-repository head
 library
   exposed-modules:     Database.PostgreSQL.TransactionalStore
   hs-source-dirs: src
-  ghc-options: -Wall -Werror
+  ghc-options: -Wall
```

Comment:
Just a heads up: `stack sdist` whines at you if you have `-Werror` on, with the justification that new versions of GHC can produce new warnings that will make builds fail.